modules/dxkultureBidAdapter.js

```diff
@@ -329,7 +329,7 @@ function buildVideoRequestData(bidRequest, bidderRequest) {
 
   // Placement Inference Rules:
   // - If no placement is defined then default to 1 (In Stream)
-  video.placement = video.placement || 2;
+  video.placement = video.placement || 1;
```

Comment:
video.placement is deprecated; why not also support plcmt? The reason for the deprecation was actually logic like this; the buy side described instream as over-specified. 

Also pls cover in unit tests

Reply:
@patmmccann We are still operating on openrtb 2.5. We made suggested changes